### PR TITLE
Add exclusion file

### PR DESCRIPTION
**Pull request by karstenpatzwaldt**
Add the option to specify a file with site IDs to ignore. This is useful to filter out things like group campsites.

### CODE REVIEW

File: camping.py [hunk 1]
```diff
@@ -263,7 +266,28 @@ def generate_json_output(info_by_park_id):
...
 
+def remove_comments(lines: list[str]) -> list[str]:
+    new_lines = []
+    for line in lines:
+        if line.startswith("#"):  # Deal with comment as the first character
+            continue
+
+        line = line.split(" #")[0]
+        if line.strip() != "":
+            new_lines.append(line)
```
Comment (banool) [inline camping.py:277]: Should you use `line.strip()` here too?

File: camping.py [hunk 2]
```diff
@@ -4,6 +4,7 @@
 import json
 import logging
 import sys
+import re
```
Comment (banool) [inline camping.py:7]: Is this used?

**Review by banool — CHANGES_REQUESTED** [review]
Thanks a lot for this! Just a couple of comments.

**Comment by karstenpatzwaldt** [comment]
Addressed your ocmments

**Comment by banool** [comment]
Thanks a lot!!

**Pull request merged**